fonts/manifest.yml

```diff
@@ -35,31 +35,47 @@ Cambria Math:
   - Regular
 
 Work Sans:
-  - Black
-  - Black Italic
+  - Regular
   - Bold
+  - Italic
   - Bold Italic
-  - ExtraBold
-  - ExtraBold Italic
-  - ExtraLight
-  - ExtraLight Italic
+
+Work Sans Black:
+  - Regular
   - Italic
-  - Light
-  - Light Italic
-  - Medium
-  - Medium Italic
+
+Work Sans ExtraBold:
   - Regular
-  - SemiBold
-  - SemiBold Italic
-  - Thin
-  - Thin Italic
- 
+  - Italic
+
+Work Sans ExtraLight:
+  - Regular
+  - Italic
+
+Work Sans Light:
+  - Regular
+  - Italic
+
+Work Sans Medium:
+  - Regular
+  - Italic
+
+Work Sans SemiBold:
+  - Regular
+  - Italic
+
+Work Sans Thin:
+  - Regular
+  - Italic
+
 STIX Two Math:
   - Regular
 
 Source Han Sans:
   - Bold
-  - Normal
+  - Regular
+
+Source Han Sans Normal:
   - Regular

```

Comment:
@Intelligent2013 Seems I need help with debugging. By some reason it throws this error, though a path for "Source Han Sans Normal" is returned from fontist.

```
Transforming...
java.io.IOException: Can't read /home/runner/.metanorma/fonts/SourceHanSans-Normal.ttc
	at java.desktop/java.awt.Font.checkFontFile(Font.java:1244)
	at java.desktop/java.awt.Font.createFont(Font.java:1224)
	at org.metanorma.fop.fontConfig.updateFontsForGraphicsEnvironment(fontConfig.java:688)
```

Reply:
@alexeymorozov I have to update mn2pdf for a new fontist format.
Before:
```
Work Sans:
  - Black
  - Black Italic
  - ...
```
New:
```
Work Sans Black:
  - Regular
  - Italic
 ```

Reply:
Just FYI, mn2pdf updated (v1.35b release for testing purposes only) for support both fontist formats (tested here https://github.com/metanorma/mn-native-pdf/pull/324).

Reply:
Thanks :) Let's rerun the build then